### PR TITLE
DEV-3187-new-go-exercise-revconcatalternate

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

File: subjects/revconcatalternate/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+## Reverse-Concat-Alternate
+
+
+### Instructions
+
+- Write a function that receives two slices of int as arguments and returns a new one with the result of alternating the values of each in reverse order.
```
Comment (zainabdnaya) [inline subjects/revconcatalternate/README.md:6]: It's better if remove `-` form first line

File: subjects/revconcatalternate/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println(RevConcatAlternate([]int{1,2,3},[]int{4,5,6}))
```
Comment (zainabdnaya) [inline subjects/revconcatalternate/README.md:32]: could you remove space here

File: subjects/revconcatalternate/README.md (hunk 1 above)
Comment (Hamzaelkhatri) [reply subjects/revconcatalternate/README.md]: fixed

File: subjects/revconcatalternate/README.md (hunk 2 above)
Comment (Hamzaelkhatri) [reply subjects/revconcatalternate/README.md:32]: fixed

File: subjects/revconcatalternate/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+## Reverse-Concat-Alternate
```
Comment (zainabdnaya) [inline subjects/revconcatalternate/README.md:1]: This should be lower case

Comment (Hamzaelkhatri) [reply subjects/revconcatalternate/README.md]: changed

**Review by zainabdnaya — APPROVED** [review]

File: subjects/revconcatalternate/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+## revconcatalternate
+
+### Instructions
+
+Write a function `RevConcatAlternate()` that receives two slices of `int` as arguments and returns a new slice with alternated values of each slice in reverse order.
+- The input slices can have different lengths.
+- The new slice should contain the elements of the larger slice first and then the elements of the smaller slice.
```
Comment (jotapero) [inline subjects/revconcatalternate/README.md:7]: fix: The new slice should start with an element of the largest slice first.

File: subjects/revconcatalternate/README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+- If the slices are of equal length, the new slice should contain the elements of the first slice first and then the elements of the second slice.
```
Comment (jotapero) [inline subjects/revconcatalternate/README.md:8]: fix: (...) should start with an element of the first slice.

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/revconcatalternate/README.md (hunk 4 above)
Comment (tiagocollot) [reply subjects/revconcatalternate/README.md:7]: Thank you for your correction. It is fixed!

File: subjects/revconcatalternate/README.md (hunk 5 above)
Comment (tiagocollot) [reply subjects/revconcatalternate/README.md:8]: It is fixed.

**Review by jotapero — APPROVED** [review]
LGTM :alien:

File: subjects/revconcatalternate/README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+Write a function `RevConcatAlternate()` that receives two slices of `int` as arguments and returns a new slice with alternated values of each slice in reverse order.
+- The input slices can have different lengths.
+- The new slice should start with an element of the largest slice first.
+- If the slices are of equal length, the new slice should start with an element of the first slice.
+
+### Expected function
+
+```go
+func RevConcatAlternate(slice1,slice2 []int) []int {
```
Comment (eslopfer) [inline subjects/revconcatalternate/README.md:13]: Please add a space between parameters.

File: subjects/revconcatalternate/README.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
+## revconcatalternate
+
+### Instructions
+
+Write a function `RevConcatAlternate()` that receives two slices of `int` as arguments and returns a new slice with alternated values of each slice in reverse order.
```
Comment (eslopfer) [inline subjects/revconcatalternate/README.md:5]: When I ran prettier it complained here. Please run it before asking for the review.

**Review by eslopfer — DISMISSED** [review]

**Pull request merged**